### ISSUE TITLE
How to rotate screen?

### ISSUE THREAD

**Issue by kanocz** [yunit-io/yunit#18]
On some devices (like GPD Pocket) screen is rotated and this solved by rotation in Xrandr or similar. Can't found how to set manual screen rotation in unity8/yunit. Is it possible?

**Comment by jsalatas** [comment]
I don't think it is possible through xrandr. Maybe Mir has a similar utility or something? :\

**Comment by kanocz** [comment]
@jsalatas it's not possible through xrandr and I can't find any information about existence some similar utility for Mir, that's why I post question here :)

**Comment by jsalatas** [comment]
Maybe @AlanGriffiths could provide some feedback here?

**Comment by AlanGriffiths** [comment]
I'm not sure of the state in Yunit (because I'm not sure where Unity8 got to), but Mir has a utility `mirout` that can rotate the display.

```
$ mirout -h
Usage: mirout [OPTIONS] [/path/to/mir/socket] [[output OUTPUTID] ACTION ...]
Options:
    -h       Show this help information.
    -i PATH  Load display configuration from a file instead of the server.
    -o PATH  Save resulting configuration to a file.
    -v       Show verbose information.
    --       Ignore the rest of the command line.
Actions:
    off | suspend | standby | on
    disable | enable
    rotate (normal | inverted | left | right)
    place +X+Y
    mode (WIDTHxHEIGHT | preferred) [rate HZ]
    fakemode WIDTHxHEIGHT
    rate HZ
```
So, something like:
`
$ mirout output 1 rotate left`

**Comment by jsalatas** [comment]
Now that you mention it, there are some ui unit tests which rotate the screen. So maybe I guess the functionality is there but maybe not exposed to the user interface.

**Comment by kanocz** [comment]
@AlanGriffiths got black screen after this command, but thank you anyway - now I have a point to start experiments! :)

**Comment by jsalatas** [comment]
@kanocz just a crazy thought: are you trying on physical hardware or in virtual machine?

**Comment by kanocz** [comment]
@jsalatas on physical hardware

**Comment by kanocz** [comment]
ah... it's not a black screen, it's mir server crash :D
